main.js

```diff
@@ -160,4 +160,42 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+// Opdracht 1a
+const tvTypes = inventory.map((stock) => {
+  return stock.type;
+});
+
+console.log(tvTypes);
+
+// Opdracht 1b
+const SoldOutTvs = inventory.filter((stock) => {
+  if(stock.originalStock - stock.sold === 0){
+    return stock;
+  }
```

Comment:
De filter functie verwacht een boolean terug te krijgen. Daarnaast kun je dit ook een stuk simpeler doen
```suggestion
  return stock.originalStock === stock.sold;
```